src/cloud/docker/docker-mftf.md

```diff
@@ -5,12 +5,14 @@ functional_areas:
   - Cloud
 ---
 
-In a Docker development environment, you can use the [Magento Functional Testing Framework] (MFTF)]{{site.baseurl}}/mftf/docs/introduction.html)(M) for Magento application testing.
+In a Cloud Docker development environment, you can use the [Magento Functional Testing Framework (MFTF)][MFTF docs] for Magento application testing.
+
+You use the [CLI container command] `mftf-command` to run MFTF commands in the Cloud Docker environment, for example `docker-compose run test mftf-command generate:tests --debug=none`.
```

Comment:
This is good to have! Could use a little more comment about what the example is doing, though.